asab/metrics/metrics.py

```diff
@@ -102,6 +103,33 @@ def rest_get(self):
 		return rest
 
 
+class EPSCounter(Counter):
```

Comment:
"""
Event per Second Counter
"""

... providing a bit of docstring is helpful here.